### ISSUE TITLE
請問如何重現EncNet的結果

### ISSUE THREAD

**Issue by henryace** [wuhuikai/FastFCN#16]
你好，請問一下如何重現EncNet的結果

github上只有EncNet+JPU的script而無EncNet的script

想重現EncNet的結果將 --jpu --aux 拿掉就可以了嗎?

還是有其他地方要改?

Thanks !

**Comment by wuhuikai** [comment]
```
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --dataset pcontext \
    --model encnet --dilated --aux --se-loss \
    --backbone resnet50 --checkname encnet_res50_pcontext
```

**Comment by HolmesShuan** [comment]
Sorry to bother you again. I wonder how to reproduce the results of DeepLabv3 and PSPNet, reported in Table 1. I use the following code to train DeepLabv3, but it converges pretty slow:
```shell
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --dataset pcontext \
    --model deeplab --aux --se-loss\
    --backbone resnet50 --checkname deeplab_res50_pcontext --workers 8 --no-val
```

**Comment by wuhuikai** [comment]
@HolmesShuan Training for 80 epochs is enough, which should finish within 1 day on 4 Titan-Xp GPUs.

**Comment by HolmesShuan** [comment]
Thanks for your responses, and should `--se-loss` be used in ASPP and PSPNet?

**Comment by wuhuikai** [comment]
@HolmesShuan No, it's used in `EncNet` only.

**Comment by henry8527** [comment]
may I ask that for reproducing original EncNet, when to use the args['lateral'] ?

**Comment by wuhuikai** [comment]
The official code shows that they use `lateral` when the dataset is `Pascal`. ([link](https://github.com/zhanghang1989/PyTorch-Encoding/blob/ae218bf96115b613732a2be40b0fbd3ee15b7a7c/encoding/models/encnet.py#L137))

**Comment by henry8527** [comment]
Thanks, but why your command didn't use --lateral? 
Since it's also using Pascal family dataset (Pascal-Context).
https://github.com/wuhuikai/FastFCN/issues/16#issuecomment-486943216

**Comment by wuhuikai** [comment]
Ops, I forgot lateral in the above command.

**Comment by henry8527** [comment]
How about the correct script for reproducing PSPNet baseline without jpu?  thanks a lot.

**Comment by wuhuikai** [comment]
```
CUDA_VISIBLE_DEVICES=0,1,2,3 python train.py --dataset pcontext \
    --model psp --dilated --aux \
    --backbone resnet50 --checkname psp_res50_pcontext
```
